.github/build.sh

```diff
@@ -233,3 +233,4 @@ cmake --build . --target test-aws-iot-device-client
 
 ### Run Tests ###
 env AWS_CRT_MEMORY_TRACING=1 ./test/test-aws-iot-device-client
+./test/test-aws-iot-device-client --gtest_filter=LogQueueTest.notifyAllOnShutdown --gtest_repeat=1000
```

Comment:
Keep this in for now, but comment this out in the future after we have confidence that the issue is resolved.